core/src/main/java/hudson/model/Run.java

```diff
@@ -157,6 +157,13 @@
      */
     public static final long QUEUE_ID_UNKNOWN = -1;
 
+    /**
+     * Target size limit for truncated {@link #description}s in the Build History Widget.
```

Comment:
```suggestion
     * Target size limit for truncated {@link #description}s in the Build History Widget.
     * This is applied to the raw, unformatted description. Especially complex formatting
     * like hyperlinks can result in much less text being shown than this might imply.
```